src/libraries/System.Security.Cryptography/src/System.Security.Cryptography.csproj

```diff
@@ -1,12 +1,41 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent);$(NetCoreAppCurrent)-Android;$(NetCoreAppCurrent)-OSX;$(NetCoreAppCurrent)-iOS;$(NetCoreAppCurrent)-tvOS</TargetFrameworks>
```

Comment:
Can we put the OS-less `$(NetCoreAppCurrent)` at the front or the end of the string? That way is simpler to read/find?

Reply:
I copied this as-was from Encoding.  I think it used to be last then some more OS variants got added more-last.

First would be stable, but VS Solution Explorer only reliably shows files conditionally included from the first target framework, which is the least useful here (since the AnyOS build is a generated PNSE).